rotkehlchen/tests/unit/test_eth2.py

```diff
@@ -1,191 +1,387 @@
-            validator_index=1767,
-            tx_hash='0x9c199a1a570e5c0e5efcfa8bd0b2fbffc0e7847b72680dee161de15d57bc9eef',
-            log_index=244,
-        ), Eth2Deposit(
-            from_address=addr2,
-            pubkey='0x8209a3cd141f3ccc9e60df64a76334c1306f857b471ef4ad50eca889f2e4f8e03ae24f5c48a07b53266d373c8929fc37',  # noqa: E501
-            withdrawal_credentials='0x005c1ba1222d4b6ba8fa7c77ecefbc33fde0dd69127bb00189e247a8febdce13',  # noqa: E501
-            value=Balance(FVal(32), FVal(64)),
-            validator_index=1768,
-            tx_hash='0x8cfb18653dde780b922c2f2a1e45314b7613810b35a46ab15f630c7577eb3f16',
-            log_index=299,
-        ), Eth2Deposit(
-            from_address=addr2,
-            pubkey='0xb0e6c52bc96ea0574edfb9f1c5350a85d28113d418af6accc53c0bb14407be68ecc426a5208a882859068758643a2e8c',  # noqa: E501
-            withdrawal_credentials='0x004767c06d7caf85f970c9f62e3ca4d02392ccdccc97e90abe858604d6779a01',  # noqa: E501
-            value=Balance(FVal(32), FVal(64)),
-            validator_index=1769,
-            tx_hash='0x91b77d3c38b272af7c236903370bbc26ed5a23ff4d59e61762ea13a763a959ec',
-            log_index=229,
-        ), Eth2Deposit(
-            from_address=addr2,
-            pubkey='0xa96f99e00213be6b0c9c7491589177e7406f811d7b687e14ed68bc51c713859fd44d35360b9c84cb4479f538b94b83d6',  # noqa: E501
-            withdrawal_credentials='0x00c35e1ba0626bcb4ca2ca6ffd1ae44816578d2a6e6cbdb8e98251cd4ab09f7c',  # noqa: E501
-            value=Balance(FVal(32), FVal(64)),
-            validator_index=1770,
-            tx_hash='0x82ac4855470afc09dd498b59affe7de755ac3ae5fad75d49b627d0c0f31e4d8e',
-            log_index=265,
-        ), Eth2Deposit(
-            from_address=addr2,
-            pubkey='0x8baf860d88a3c3bb2d5228680782242294b50378ded9988698b98c478f45e895047e6399a953e918bb47821e8debe031',  # noqa: E501
-            withdrawal_credentials='0x00a74306e6546011277c9d97477a991b8c02b465b5eae65faaa5682258a8e0cf',  # noqa: E501
-            value=Balance(FVal(32), FVal(64)),
-            validator_index=1771,
-            tx_hash='0x5b64fbf67b652b446ce9dabc084cd7766cc15325afd79c0525dab9d29cce250c',
-            log_index=257,
-        ),
+    assert total_addr1 == FVal(32)
+    assert total_addr2 == FVal(64)
+    assert total_addr3 == FVal(480)
+
+
+@pytest.mark.freeze_time(datetime(2020, 11, 4, 16, 18, 5))
+@pytest.mark.parametrize(*ETHEREUM_TEST_PARAMETERS)
+@pytest.mark.parametrize('default_mock_price_value', [FVal(2)])
+def test_get_eth2_staked_amount_fetch_from_db(  # pylint: disable=unused-argument
+        ethereum_manager,
+        call_order,
+        ethereum_manager_connect_at_start,
+        inquirer,
+        price_historian,
+        freezer,
+):
+    """
+    Test new on-chain requests for existing addresses start from last used
+    query range `end_ts`, to "now".
+
+    For avoiding request since ETH2 deposit contract deployment timestamp
```

Comment:
I don't think what you write here makes sense.

Do you simply want to say that you want to freeze `now` to be `1604506685` ?